### DISCUSSION TITLE
Chainlink subscription

### DISCUSSION THREAD

**Discussion by radin100** [Cyfrin/foundry-full-course-cu#1103] [Q&A] [answered]
My questions are why I need a subscription to use some of Chainlink functions that we use in lesson 9? New is when I make it, is that a subscription from my whole MeatMask wallet to chainlink?
And why I need to fund it with different currency?

**Comment by 0xusmanf** [comment, marked as answer]
> My questions are why I need a subscription to use some of Chainlink functions that we use in lesson 9? New is when I make it, is that a subscription from my whole MeatMask wallet to chainlink?
> And why I need to fund it with different currency?

You need a subscription because in order to use the Chainlink services you have to pay them (not all Chainlink services are free like Price Feeds). 

I am not sure what you mean by whole Metamask account but you are subscribing to it through the account you connect to the Chainlink service.

You have to pay them with $LINK token because it's their native token for the Chainlink ecosystem and they use it to charge you for their service used and that's the only token they accept as payment.
